plugin/pkg/admission/persistentvolume/resize/admission.go

```diff
@@ -98,21 +98,31 @@ func (pvcr *persistentVolumeClaimResize) Admit(a admission.Attributes) error {
 		return nil
 	}
 
-	// Growing Persistent volumes is only allowed for PVCs for which their StorageClass
-	// explicitly allows it
-	if !pvcr.allowResize(pvc, oldPvc) {
-		return admission.NewForbidden(a, fmt.Errorf("only dynamically provisioned pvc can be resized and "+
-			"the storageclass that provisions the pvc must support resize"))
-	}
-
-	// volume plugin must support resize
-	pv, err := pvcr.pvLister.Get(pvc.Spec.VolumeName)
-	if err != nil {
-		return nil
-	}
+	oldSize := oldPvc.Spec.Resources.Requests[api.ResourceStorage]
+	newSize := pvc.Spec.Resources.Requests[api.ResourceStorage]
+
+	if newSize.Cmp(oldSize) > 0 {
```

Comment:
return early and unnest:

```
if newSize.Cmp(oldSize) <= 0 {
  return nil
}
```

Reply:
I think the current `if` as it exists conveys the intention better.  let me know if this is blocker for you though.

Reply:
returning early rather than indenting large blocks of a function is preferred by kube and go coding conventions

Reply:
ack done.